src/axom/sidre/core/View.hpp

```diff
@@ -929,12 +929,24 @@ class View
   //@{
   //!  @name Attribute Value query and accessor methods
 
+  /*!
+   * \return the index-specified Attribute, or nullptr if it doesn't exist.
+   */
   Attribute* getAttribute(IndexType idx);
 
+  /*!
+   * \return the index-specified Attribute, or nullptr if it doesn't exist.
```

Comment:
Can you add a `brief` to these? also you should be able to just list them as overloads instead of copy/pasting.

Reply:
Will do.

Reply:
@gunney1 you can put the common behavior of the getAttribute() methods, etc. in the comment block starting at line 930, where methods are grouped if you want.